Git/README.md

```diff
@@ -20,39 +19,55 @@ git config --global user.email "your_email@youremail.com"
 
 They will get added to your `.gitconfig` file.
 
-To push code to your GitHub repositories, we're going to use the recommended
-HTTPS method (versus SSH). To prevent `git` from asking for your username and
-password every time you push a commit you can cache your credentials by running
-the following command, as described in the
-[instructions](https://help.github.com/articles/caching-your-github-password-in-git/).
+To push code to your GitHub repositories, we will use the recommended HTTPS method. There are also instructions for using SSH. To prevent `git` from asking for your username and password every time you push a commit you can cache your credentials by running the following command, as described in the [instructions](https://help.github.com/articles/caching-your-github-password-in-git/).
 
-    git config --global credential.helper osxkeychain
+```sh
+git config --global credential.helper osxkeychain
+```
+
+## Using HTTPS for GitHub (recommended)
+
+The instructions below are referenced from [the official documentation](https://help.github.com/en/github/using-git/which-remote-url-should-i-use#cloning-with-https-urls-recommended).
+
+### Clone repositories using HTTPS
+
+After creating a new repo on GitHub, clone it using
+
+```sh
+git clone https://github.com/<username>/<repo-name>.git
+```
+
+If you initialized with a README
+
+If you did not, follow the instructions in the section below
```

Comment:
Missing a period or colon at the end of this sentence?

Reply:
```suggestion
If you did not, follow the instructions in the section below.
```